tools/cldr-apps/js/src/esm/cldrLoad.js

```diff
@@ -34,6 +34,7 @@ import * as cldrSurvey from "./cldrSurvey.js";
 import * as cldrTable from "./cldrTable.js";
 import * as cldrText from "./cldrText.js";
 import * as cldrVettingParticipation from "./cldrVettingParticipation.js";
+import { isVueSpecial } from "../specialToComponentMap";
 
```

Comment:
I would include the ".js" explicitly -- more compatible, less webpack-dependent

also let's try to use mostly js in src/esm, minimize js directly in src

imports could be better organized if they were more consistent